sql/src/main/java/io/crate/execution/engine/collect/sources/ShardCollectSource.java

```diff
@@ -299,23 +299,34 @@ public CrateCollector getCollector(CollectPhase phase,
             );
         }
 
-        RowConsumer firstConsumer = ProjectingRowConsumer.create(
-            lastConsumer,
+        Projectors projectors = new Projectors(
             normalizedPhase.projections(),
-            collectPhase.jobId(),
+            normalizedPhase.jobId(),
```

Comment:
what's the reason for this change?

Reply:
No real reason other that my OCD didn't like having `normalizedPhase` followed by `collectPhase`, given that both have the same `jobId`